charts/intel-resource-drivers/README.md

```diff
@@ -9,37 +9,34 @@ helm repo update
 You can execute `helm search repo intel` command to see pulled charts [optional].
 
 ## Install Helm Chart
```

Comment:
Please add documentation how to install just one resource driver, not all of them at once.

Reply:
It's worth to mention that none of the resource drivers are installed by default and point to table of values to be set for installing particular driver below.